pkg/agent/openflow/pipeline_windows.go

```diff
@@ -311,3 +311,54 @@ func (c *client) l3FwdFlowToRemoteViaRouting(localGatewayMAC net.HardwareAddr, r
 	}
 	return []binding.Flow{c.l3FwdFlowToRemoteViaGW(localGatewayMAC, *peerPodCIDR, category)}
 }
+
+func (c *client) arpResponderClusterIPFlows(svcIP net.IP, gwMAC net.HardwareAddr, category cookie.Category) (flows []binding.Flow) {
+	flows = append(flows, c.pipeline[arpResponderTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolARP).
+		MatchARPTpa(svcIP).MatchARPOp(arpOpRequest).
+		Action().LoadARPOperation(arpOpReply).
+		Action().Move(binding.NxmFieldSrcMAC, binding.NxmFieldDstMAC).
+		Action().SetSrcMAC(gwMAC).
+		Action().Move(binding.NxmFieldARPSha, binding.NxmFieldARPTha).
+		Action().SetARPSha(gwMAC).
+		Action().Move(binding.NxmFieldARPSpa, binding.NxmFieldARPTpa).
+		Action().SetARPSpa(svcIP).
+		Action().OutputInPort().
+		Cookie(c.cookieAllocator.Request(category).Raw()).
+		Done())
+	return flows
+}
+
+func (c *client) clusterIPHostNetworkFlows(gwIP net.IP, nodeIP net.IP, category cookie.Category) (flows []binding.Flow) {
+	flows = append(flows,
+		c.pipeline[l3ForwardingTable].BuildFlow(priorityLow).MatchProtocol(binding.ProtocolIP).
+			MatchSrcIP(gwIP).MatchCTMark(ServiceCTMark, nil).
+			MatchRegRange(int(marksReg), snatDefaultMark, endpointPortRegRange).
+			Action().LoadRegRange(int(marksReg), snatDefaultMark, snatMarkRange).
+			Action().ResubmitToTable(l2ForwardingCalcTable).
+			Cookie(c.cookieAllocator.Request(category).Raw()).
+			Done(),
+		c.pipeline[conntrackCommitTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+			MatchSrcIP(gwIP).MatchCTMark(ServiceCTMark, nil).
+			MatchCTStateNew(true).MatchCTStateTrk(true).
+			MatchRegRange(int(marksReg), snatDefaultMark, snatMarkRange).
+			Action().CT(true, L2ForwardingOutTable, ctZoneSNAT).SNAT(
+			&binding.IPRange{StartIP: nodeIP, EndIP: nodeIP},
+			nil).
+			LoadToMark(snatCTMark).CTDone().
+			Cookie(c.cookieAllocator.Request(category).Raw()).
+			Done(),
+		c.pipeline[conntrackCommitTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+			MatchSrcIP(gwIP).MatchCTMark(ServiceCTMark, nil).
+			MatchCTStateNew(false).MatchCTStateTrk(true).
+			MatchRegRange(int(marksReg), snatDefaultMark, snatMarkRange).
+			Action().CT(false, L2ForwardingOutTable, ctZoneSNAT).NAT().CTDone().
+			Cookie(c.cookieAllocator.Request(category).Raw()).
+			Done(),
+		c.pipeline[L2ForwardingOutTable].BuildFlow(priorityHigh).MatchProtocol(binding.ProtocolIP).
+			MatchRegRange(int(marksReg), snatDefaultMark, snatMarkRange).
+			MatchRegRange(int(PortCacheReg), 0x2, endpointPortRegRange).
```

Comment:
ditto, endpointPortRegRange not making sense here

Reply:
Used another match condition.